aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -116,11 +132,41 @@ spec aptos_framework::account {
         account_public_key_bytes: vector<u8>,
         recipient_address: address
     ) {
-        // TODO: complex aborts conditions.
-        pragma aborts_if_is_partial;
         let source_address = signer::address_of(account);
+        let account_resource = global<Account>(source_address);
```

Comment:
This spec also looks correct, but I still have the same modularity concerns [from above](https://github.com/aptos-labs/aptos-core/pull/5832/files#r1046629266)

Reply:
@alinush , I made this modular too. Please let me know what you think.